conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -50,5 +52,11 @@ conda build /recipe_root --quiet || exit 1
 {%- endfor -%}
 {%- endblock -%}
 
+touch /feedstock_root/build_artefacts/conda-forge-build-done
 EOF
 
+# double-check that the build got to the end
+# see https://github.com/conda-forge/conda-smithy/pull/337
+# for a possible fix
+set -x
+test -f "$FEEDSTOCK_ROOT/build_artefacts/conda-forge-build-done" || exit 1
```

Comment:
An empty line is added after here in PR ( https://github.com/conda-forge/conda-smithy/pull/459 ) so the re-rendered file has a terminal newline.